pyaedt/modules/Material.py

```diff
@@ -675,6 +685,38 @@ def __init__(self, parent, name, props=None):
                                                             MatProperties.get_defaultvalue(aedtname=property), None)
         pass
 
+    @property
+    def material_appearance(self):
+        """Material Appearance specified as an RGB list.
+
+        Returns
+        -------
+        type
+            Appearance of the material.
+        """
+        return self._material_appearance
+
+    @material_appearance.setter
+    def material_appearance(self, value):
+        if type(value) is list and len(value) == 3:
+            value_int = []
+            for i in value:
+                try:
+                    tmp = int(i)
+                except ValueError:
+                    raise
+                if tmp < 0 or tmp > 255:
+                    raise ValueError
+                value_int.append(tmp)
+            self._material_appearance = value_int
+            self._props["AttachedData"] = OrderedDict({"MatAppearanceData":
+                                                       OrderedDict({'property_data': 'appearance_data',
+                                                                    'Red': value_int[0],
+                                                                    'Green': value_int[1],
+                                                                    'Blue': value_int[2]})})
+        else:
+            raise ValueError
```

Comment:
You will have to deindent this section.  With setters, it's best to do type checking and raise early rather than branching.

Reply:
Already done it.